.github/workflows/ci.ml

```diff
@@ -15,7 +15,10 @@ open Lib
 
 let ocamls = [
   (* Fully supported versions *)
-   "4.08.1"; "4.09.1"; "4.10.2"; "4.11.2"; "4.12.1"; "4.13.1"; "5.0.0"; "5.1.1"; "4.14.1";
+   "4.08.1"; "4.09.1"; "4.10.2"; "4.11.2"; "4.12.1"; "4.13.1";
+   "5.0.0"; "5.1.1";
+   "4.14.1"; "5.2.0"; (* The last elements of the list after 4.14 be used as
```

Comment:
oops i forgot MSVC is only supported by 5.3 not 5.2. It might be worth waiting for 5.3.0~alpha1